Require two approvals from different companies before merging the PR (#364)
open-telemetry/semantic-conventions: CONTRIBUTING.md

make misspell-correction
```

### How to get your PR merged

A PR (pull request) is considered to be **ready to merge** when:

> * It has received more than two approvals from the [code

Reviewer: `... more than two approvals ...`: do we want at least two or at least three? With the current phrasing it's the latter and not consistent with what is setup in the GitHub settings for this repo as the min requirement:

<img width="330" alt="image" src="https://github.com/open-telemetry/semantic-conventions/assets/866830/7f29018d-cab6-483e-af5b-27174a5efda1">

Author: Good catch! I've updated the wording (the intention was to have at least two valid approvals).

> * It has received more than two approvals from the [code

Author: ```suggestion
* It has received at least two approvals from the [code
```
